.github/workflows/build-and-publish-catalog-on-release.yaml

```diff
@@ -21,8 +21,36 @@ jobs:
       - name: Run tests
         uses: ./.github/actions/lint
 
+  check-version:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Check package version
+        env:
+          TAG_VERSION: ${{ github.event.release.tag_name }}
+        run: |
+          VERSION=$(jq -r .version package.json)
+          EXTENSION_VERSION=$(jq -r .version pkg/harvester/package.json)
+          PKG_VERSION="v${EXTENSION_VERSION}"
+
+          if [[ "$VERSION" != "$EXTENSION_VERSION" ]]; then
+            echo "Package version mismatch: $VERSION vs $EXTENSION_VERSION"
+            exit 1
+          fi
+
+          if [[ "${TAG_VERSION}" != "$PKG_VERSION" ]]; then
+            echo "Package version $PKG_VERSION in pkg/harvester/package.json does not match tag version ${TAG_VERSION}"
+            exit 1
+          fi
+         
+          echo "Package version and tag version match: ${TAG_VERSION}"
+
   build-and-push-extension-catalog:
-    needs: lint
+    needs: 
+      - lint
```

Comment:
Please help remove the lint step from these two files, as discussed earlier, we don’t need it during the build stage.